.gitignore

```diff
@@ -64,7 +64,6 @@ target/
 # lock file is not needed for development
 # as project supports variety of Django versions
 poetry.lock
-pyproject.toml
```

Comment:
Noticed that this was in the gitignore. Strangely, I don't think it made the top-level `pyproject.toml` ignored. Since the project now uses `pyproject.toml`, we should remove this one.